scheduler/plugin/plugin_test.go

```diff
@@ -170,9 +194,14 @@ func TestNewFromConfig(t *testing.T) {
 			name:      "valid wasm",
 			guestPath: test.PathExampleFilterSimple,
 		},
+		{
+			name:          "not plugin",
+			guestPath:     test.PathErrorNotPlugin,
+			expectedError: `wasm: guest does not export any plugin functions`,
```

Comment:
now we show how something that is wasm, but not a plugin shows up. Basically it is an error not export support for at least one plugin type.